Documentation/ApiOverview/DependencyInjection/Index.rst

```diff
@@ -476,13 +476,17 @@ autoconfigure
     available from the Symfony container and marked as shared
     (:yaml:`shared: true`).
 
-Model exclusion
+.. _dependency-injection-exclusion:
+
+Classpath Exclusion
     The path exclusion :yaml:`exclude: '../Classes/Domain/Model/*'` excludes
     your models from the dependency injection container, which means you cannot inject them
     nor inject dependencies into them. Models are not services and therefore
     should not require dependency injection. Also, these objects are created by
     the Extbase persistence layer, which does not support the DI container.
-
+    There are other possible error cases with classes called from outside of an extension
```

Comment:
A scenario of the usage of a class method with dependency injection does not need such an exclude. It is only needed in an error case where the programmer has no time for recoding of the application.

Reply:
"has no time" -> building up technical debt which **will** make it harder in the future to deal with that.

Actually, I am not convinced that this is an improvement: When you have an "error case" you should solve this error and not circumvent/building around that error. So I am also against using the word "scenario",

Regarding the exclude example: I am in favor of a sentence like the [Symfony docs](https://symfony.com/doc/current/service_container.html#service-container-services-load-example) have:

> The value of the resource and exclude options can be any valid 
> [glob pattern] (https://en.wikipedia.org/wiki/Glob_(programming)). The value of the exclude
> option can also be an array of glob patterns.

Reply:
How should I solve an error case with Dependendy Injection without the exclude? 
This is completely impossible to me. I guess I must use several months in code investigations to do so.  I do not have this time and means.

Reply:
You can do what you want in your code, actually, I don't care. But what I care about is, that the official documentation should not hint users to use bad practises.

Reply:
Please explain me the good way when DI throws always an exception! Are you the expert who can master this easily?

Reply:
I don't know your code. But I never had problems in the past that I had to exclude classes because of errors - only because I want them not to be available for injection by intention.